src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -235,11 +231,17 @@
     <Copy SourceFiles="@(IncludedDacpacReferenceFiles)" DestinationFolder="$(TargetDir)" />
 
     <!-- Copy any SQL artifacts created from the command line tool -->
+    <CreateItem Include="$(IntermediateOutputPath)*.sql">
+        <Output TaskParameter="Include" ItemName="SqlArtifacts" />
+    </CreateItem>
     <Copy SourceFiles="@(SqlArtifacts)" DestinationFolder="$(TargetDir)" />
   </Target>
 
   <!--Ensure that any SQL artifacts generated by the compile are cleaned up properly -->
   <Target Name="CleanSqlArtifacts" AfterTargets="AfterClean">
+      <CreateItem Include="$(IntermediateOutputPath)*.sql">
```

Comment:
Is this block meant to appear here? It's a duplicate of the block above, and we have both a `CreateItem` and a `Delete`, so I just wanted to confirm.

Reply:
Well, I guess I think that if we need the use the CreateItem to pick up the SQL to copy, then we may also need to use it to pick up SQL to delete.

I suppose another way is to keep the itemgroup as-is but just _update_ it after the create script creates the sql

Reply:
I tried various combinations but the createitem seems to be the most reliable when it comes to (a) picking up any new artifacts created by build and (b) being run in isolation (e.g. msbuild /t:clean)